components/push/src/crypto.rs

```diff
@@ -173,6 +173,7 @@ impl Cryptography for Crypto {
         salt: Option<&str>,
         dh: Option<&str>,
     ) -> error::Result<Decrypted> {
+        rc_crypto::ensure_initialized();
```

Comment:
I'm curious, how did you determine where to add these, and can we tell whether you've added them in all the right places?

Reply:
Good question, it's mostly manual process (where are the uses of rust-ece?) combined with test failures. It's not perfect obviously but I think I got all the usages.

Reply:
What are the downside risks of forgetting one?  (Crash, silent failure, memory unsafety...?)

Reply:
You would end up with a panic as we force unwrap the `OnceCell` in `rust-ece`, which would be caught by the FFI code and transformed into a regular Kotlin/Swift exception.